api/src/game_state_server.py

```diff
@@ -43,9 +43,6 @@ def __init__(self, room_store: RoomStore):
         self._rooms = {}
         self.room_store = room_store
 
-    def valid_previous_rooms(self) -> list:
```

Comment:
This change shouldn't be a part of this PR

Reply:
If you merge the other PR first, this should go away, but in general it's bad form to include unrelated changes from another PR in a PR